src/jquery.mockjax.js

```diff
@@ -275,9 +275,17 @@
 						} else {
 							this.responseText = mockHandler.responseText;
 						}
-						if( typeof mockHandler.status === 'number' || typeof mockHandler.status === 'string' ) {
+						
+						if ($.isArray(mockHandler.status)) {
+							var min = Math.ceil(0);
```

Comment:
I don't think we need this... the `Math.ceil(0)` here is always going to be `0`  :)